firefox_puppeteer/tests/test_utils.py

```diff
@@ -0,0 +1,74 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness import FirefoxTestCase
+
+
+class TestUtils(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        # Clear all previous history and cookies.
+        self.places.remove_all_history()
+        self.marionette.delete_all_cookies()
+
+        # The test urls.
+        self.urls = [
+            'layout/mozilla_projects.html',
+            'layout/mozilla.html',
+            'layout/mozilla_mission.html',
+            'cookies/cookie_single.html'
+        ]
+        self.urls = [self.marionette.absolute_url(t) for t in self.urls]
+
+        # Open the test urls, including the single cookie setting page.
+        def load_urls():
+            with self.marionette.using_context('content'):
+                for url in self.urls:
+                    self.marionette.navigate(url)
+        self.places.wait_for_visited(self.urls, load_urls)
+
+    def tearDown(self):
+        try:
+            self.places.remove_all_history()
+            self.marionette.delete_all_cookies()
+        finally:
+            FirefoxTestCase.tearDown(self)
+
+    def get_all_urls_in_history(self):
+        return self.marionette.execute_script("""
+          let hs = Cc["@mozilla.org/browser/nav-history-service;1"]
+                   .getService(Ci.nsINavHistoryService);
+          let urls = [];
+
+          let options = hs.getNewQueryOptions();
+          options.resultType = options.RESULTS_AS_URI;
+
+          let root = hs.executeQuery(hs.getNewQuery(), options).root
+          root.containerOpen = true;
+          for (let i = 0; i < root.childCount; i++) {
+            urls.push(root.getChild(i).uri)
+          }
+          root.containerOpen = false;
+
+          return urls;
+        """)
+
+    def test_sanitize_history(self):
+
+        # Call the utils sanitize().
+        self.utils.sanitize(data_type={"history": True})
```

Comment:
Also no need for the line comments. It's better you add a single-line docstring for the test method. It will then be shown in stdout when running the tests.